V3

yashh-d opened the pull request:
Updated documentation to include sector, docket file number, and sec release number

yashh-d committed:
V3

Updated documentation to include sector, docket file number, and sec release number

AlexCatarino commented on RegalyticsRegulatoryArticle.cs:
```diff
@@ -84,6 +90,9 @@ public class RegalyticsRegulatoryArticle : BaseData
         public List<string> Agencies { get; set; }
 
+        [JsonProperty(PropertyName = "sector_type")]
+        public List<string> Sector { get; set; }
```
The sector_type is a list of key-value pair (dictionary):
```
"sector_type": [{"name": "Financial"}, {"name": "Real Estate and Construction"}],
```

```csharp
public List<Dictionary<string,string>> Sector { get; set; }
```

yashh-d replied:
Sounds good, just fixed the issues. Thanks for the links they were very helpful

AlexCatarino reviewed (changes requested):
Thank you for your contribution, @yprime7!

The tests are not compiling. Please [compile and run the unit tests](https://www.quantconnect.com/docs/v2/lean-engine/contributions/datasets/testing-data-models#03-Run-Unit-Tests) before submitting the pull request. 

The sector type is not a list of string. When the reader parse the data, it throws an exception. Please [run the demonstration algorithm](https://www.quantconnect.com/docs/v2/lean-engine/contributions/datasets/testing-data-models#02-Run-Demonstration-Algorithms).

yashh-d committed:
V3 Updates

Updated documentation to include sector, docket file number, and sec release number. Test cases Successful

AlexCatarino reviewed (approved):
Thank you, @yprime7 !
